Hide toolbar since we don't need for now

PatrickDotStar opened the pull request.

Commit:
use correct toolbar bottom offset for non iOS 11 devices

Commit:
do not add the toolbar as a subview since we don't need it for now

randomguy0815 commented on SKPhotoBrowser/SKPhotoBrowser.swift:
```diff
@@ -577,7 +577,9 @@ private extension SKPhotoBrowser {
     
     func configureToolbar() {
         toolbar = SKToolbar(frame: frameForToolbarAtOrientation(), browser: self)
-        view.addSubview(toolbar)
```
Please check for `SKPhotoBrowserOptions.displayAction` so that it would work again if we use the action sheet again.

PatrickDotStar replied:
0d67df5

Commit:
Only add the toolbar if display actions are available 

Only add the toolbar if display actions are available instead of commenting the addSubview call
fix

soberman commented:
👍
